burn-tensor/src/tests/ops/narrow.rs

```diff
@@ -0,0 +1,41 @@
+#[burn_tensor_testgen::testgen(narrow)]
+mod tests {
+    use super::*;
+    use burn_tensor::{Data, Shape, Tensor};
+
+    #[test]
+    fn test_narrow() {
+        let tensor: Tensor<TestBackend, 2> =
+            Tensor::from_data(Data::from([[1., 2., 3.], [4., 5., 6.], [7., 8., 9.]]));
+
+        let output = tensor.narrow(0, 0, 2);
+        assert_eq!(output.shape(), Shape::from([2, 3]));
```

Comment:
I think a test comparing tensor data rather than just the shape would be more reassuring and self-explaining. 

Something like
```rust
let data_expected = Data::from([[1., 2., 3.], [4., 5., 6.]]);
assert_eq!(data_expected, data_actual);
```

Also you could add a test for the other dim if you want

Reply:
Yup, forgot that, already adding it!